src/test/scala/nl.knaw.dans.easy.deposit/docs/package.scala

```diff
@@ -15,24 +15,35 @@
  */
 package nl.knaw.dans.easy.deposit
 
+import java.net.UnknownHostException
+
 import better.files.StringOps
 import javax.xml.transform.stream.StreamSource
+import javax.xml.validation.Schema
 import nl.knaw.dans.lib.logging.DebugEnhancedLogging
 
-import scala.util.Try
-import scala.xml.{ Elem, PrettyPrinter }
+import scala.util.{ Failure, Try }
+import scala.xml.{ Elem, PrettyPrinter, SAXParseException }
 
 package object docs extends DebugEnhancedLogging {
   // pretty provides friendly trouble shooting for complex XML's
   val prettyPrinter: PrettyPrinter = new scala.xml.PrettyPrinter(1024, 2)
 
+  implicit class triedSchemaExtension(val triedSchema: Try[Schema]) extends AnyVal {
+    def isAvailble: Boolean = {
+      triedSchema match {
+        case Failure(e: SAXParseException) if e.getCause.isInstanceOf[UnknownHostException] => false
```

Comment:
move and reuse? 
https://github.com/DANS-KNAW/easy-deposit-api/blob/5605609622f3af91b1c67dbd6e637821d936c9c3/src/main/scala/nl.knaw.dans.easy.deposit/docs/DDM.scala#L170-L179

Reply:
what you want to do? I'd say just keep the two. One is a private function within `DDM` and the other is just meant for testing.

Reply:
My comment would be marked as outdated when I would have placed it where it belonged. At:
https://github.com/DANS-KNAW/easy-deposit-api/blob/3336e66ec3caf2e631c5617f2eac9bd28a16f374/src/test/scala/nl.knaw.dans.easy.deposit/docs/package.scala#L41-L47